specification/repository.md

```diff
@@ -22,12 +22,10 @@ approval is granted, GDI repositories MUST NOT cut a GA release.
 
 - MUST grant `Admin` [permission
   level](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-permission-levels-for-an-organization)
-  to maintainers team
+  to maintainers team and the `gdi-instrumentation-admins` team
 - MUST grant `Write` [permission
   level](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-permission-levels-for-an-organization)
-  to approvers team and the `signalfx/docs`,
-  `signalfx/gdi-specification-approvers`
-   `signalfx/gdi-specification-maintainers` teams
+  to approvers team and the `signalfx/gdi-docs` and `signalfx/gdi-specification-maintainers` teams
```

Comment:
I think we're actually using `gdi-docs` in most/all GDI repos out there.

The `signalfx/gdi-specification-approvers` wasn't given permissions in any of the repos that I checked. I removed it both from here and CODEOWNERS, and, frankly, I'm inclined to remove the `signalfx/gdi-specification-maintainers` requirement as well. Since it hasn't been implemented almost anywhere, and hasn't been a problem, we should just drop it.